src/util/text-tracks.js

```diff
@@ -24,12 +24,28 @@ export const createCaptionsTrackIfNotExists = function(inbandTextTracks, tech, c
       // in the m3u8 for us to use
       inbandTextTracks[captionStream] = track;
     } else {
+      const captionServices = tech.options_.vhs && tech.options_.vhs.captionServices || {};
```

Comment:
This section gets used when we have caption services, but they aren't being specified in the manifests. When specified in the manifest, we will apply the properties in media groups https://github.com/videojs/http-streaming/blob/ad9aa2ef67fa90557af6afa7878d911b5f1093ac/src/media-groups.js#L641-L662
which will need an update to use this captionServices data

Reply:
Might be worth adding a comment here mentioning that.

Reply:
Yeah, makes sense. Added